docs/proposals/scenarios/privacy-preserving-llm-collaboration/optimizing-privacy-performance-equilibrium-cloud-edge-llm-systems_en_PR.md

```diff
@@ -0,0 +1,215 @@
+
+This is a simulated multi-turn conversation in a cross-border e-commerce scenario, demonstrating how the framework dynamically adjusts its privacy strategy based on the conversation content.
+
+* **Jurisdiction & Computing Power:** The edge side is located within mainland China (simulated via Colab), while the cloud side is located overseas (Ianvs inference service).  
+* **Privacy Policy Parameters:** High-sensitivity ε≤0.8, general ε≤1.2, default δ=1e-5; session-level budget accumulation and rate limiting.  
+* **Zero Raw Text Cross-Border:** Only the "anonymized vector \+ minimal necessary tags" are transmitted throughout the entire process, ensuring the original text never leaves the local jurisdiction.
+
+**Turn 1**
+
+* **User:** The iPhone 15 Pro I bought from your store last week is overheating badly. How can I return it?  
+* **Edge Side:**  
+  * The PIPL classifier identifies this prompt as **General Sensitivity** (product issue, no PII).  
+  * Applies policy: ε=1.0, saliency mask ratio 0.4, projection dimension 768→64.  
+  * **Cross-Border Payload:** Anonymized Vector (64-dim) + {"intent": "return_request"}.  
+* **Cloud Side:** Based on the anonymized vector and intent tag, it generates a response with the general return policy and procedure.
+
+**Turn 2**
+
+* **User:** The order number is 2024-09-01-3309, recipient is John Doe, phone is 138****2567. Do I need to pay for the return shipping?  
+* **Edge Side:**  
+  * The PIPL classifier identifies **High-Sensitivity Entities** (order number/name/phone).  
+  * Triggers high-sensitivity policy: ε=0.8; performs local order number validation and entity sanitization.  
+  * **Cross-Border Payload:** Anonymized Vector (64-dim) + {"order_valid": true, "return_period": "in_range"}.  
+* **Cloud Side:** Based on the anonymized context and boolean tags, it generates an explanation of the shipping fee policy without including any PII.
+
+**Auditing:** The PIPL classification, sanitization/transformation parameters (ε, mask_ratio, proj_dims), budget accumulation, and cross-border transmission hash are logged for each turn, fulfilling the requirements of minimal necessity and traceability for compliance.
+
+## **Directory & Artifacts**
+
+Project artifacts are organized as follows, covering documentation, code, evaluations, and configurations:
+
+```
+privacy_llm_cross_border/
+├── README.md
+├── configs/
+│ ├── model_config.yaml
+│ └── privacy_policy.yaml
+├── data/
+│ └── chnsenticorp_lite/
+│ ├── test.jsonl
+│ └── train.jsonl
+├── scripts/
+│ └── run_benchmark.sh
+└── src/
+├── cloud_inference.py
+├── edge_privacy.py
+└── evaluation/
+├── attacks.py
+└── metrics.py
+
+```
+
+## **Architecture & End-to-End Flow**
+
+The system adopts a strict separation of duties between the cloud and the edge to ensure the compliance of the data processing workflow.
+
+#### **System Component Architecture**
+
+![System Component Architecture](<./images/System Component Architecture.png>)
```

Comment:
DO NOT change the name of key components of KubeEdge Ianvs. 
It should be Test environment manager, test case controller and story manager. Any change should be considered in it. See example from others if you also want to add a edge-cloud one https://github.com/kubeedge/ianvs/blob/main/docs/proposals/scenarios/government_affairs/government_affairs.md